dallinger/models.py

```diff
@@ -1646,6 +1646,9 @@ def _mutated_contents(self):
             "_mutated_contents needs to be overwritten in class {}".format(type(self))
         )
 
+    def get_contents(self):
```

Comment:
I think it would be preferable in terms of simplifying the Dalligner-PsyNet relationship to write this in PsyNet: 

```py
@property
def contents(self):
    return self.answer
```

and then you don't have to implement `get_contents` in Dallinger too.

Reply:
Great suggestion, I'll push the change in PsyNet in just a bit.